server/conf/typedb.yml

```diff
@@ -45,7 +45,7 @@ log:
       level: info
       output: [ stdout, file ]
     storage:
-      filter: com.vaticle.typedb.core.rocks
+      filter: com.vaticle.typedb.core.database
       level: info # on 'debug' the server will periodically log storage properties
```

Comment:
maybe this comment should be updated to read `... log database storage properties`?